backend/audit/views.py

```diff
@@ -722,6 +722,10 @@ def post(self, request, *args, **kwargs):
             # FIXME: We should now provide a reasonable error to the user.
             if disseminated is None:
                 sac.transition_to_disseminated()
+                sac.save(
```

Comment:
I don't understand the difference between this and the transition events, or why we're doing this outside the transition methods.

Reply:
Not sure on this one. @jadudm Thoughts?